msteams-platform/sbs-gs-spfx.yml

```diff
@@ -2,7 +2,7 @@
 title: Build your first app with SPFx
 metadata:
   title: Hello World with SPFx
-  description: In this tutorial, you'll learn to build Hello World app with SPFx. The step-by-step guide install toolkit, setup development tenant, and enable sideloading.
+  description: In this tutorial, you'll learn to build Hello World app with SPFx. The step-by-step guide helps you to install toolkit, setup development tenant, and you can enable sideloading.
```

Comment:
Please read out loud with appropriate pause at commas. Does the following sound correct?

the guide helps you to
- install toolkit
- setup dev tenant
- you can enable sideloading.

Reply:
Edited and updated the description